uber/automated_emails.py

```diff
@@ -134,30 +134,32 @@ def days_before(days, dt, until=None):
 # Reminder emails for groups with a remaining balance to pay. These are sometimes necessary for when groups
 # add badges after registering.
 
```

Comment:
We decided that we'd like to have our onsite Uber configured to send emails, which will allow things like badge transfer emails to go out.  But we probably don't want to send "Remember to pay" emails or anything like that.

So I wrapped the ones we don't want sent out in an if block.